src/sagemaker/local/image.py

```diff
@@ -77,6 +77,14 @@ def __init__(self, instance_type, instance_count, image, sagemaker_session=None)
         """
         from sagemaker.local.local_session import LocalSession
 
+        # check if docker-compose is installed
+        if shutil.which('docker-compose') is None:
+            raise ImportError(
+                "'docker-compose' is not installed. Local mode feature will be broken. " +
```

Comment:
s/Local mode feature will be broken/Local Mode features will not work without docker-compose